test/test_functorch.py

```diff
@@ -22,8 +22,8 @@
 try:
     from functorch import (
         make_functional_with_buffers as functorch_make_functional_with_buffers,
-        vmap,
     )
+    from torch import vmap
```

Comment:
When testing on PyTorch<2.0 this will result in an `ImportError` and hence the tests getting skipped. So we'll lose test coverage of older versions where it would have been possible to import `vmap` from `functorch` instead.